config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -261,7 +261,64 @@ periodics:
     testgrid-tab-name: dualstack-azure-e2e-master-ipvs
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: "Dual-stack and Conformance e2e tests on a Kubernetes cluster provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud in ipvs mode"
-
+- interval: 24h
+  name: ci-dualstack-azure-e2e-master-windows
+  decorate: true
+  decoration_config:
+    timeout: 5h
+  labels:
+    preset-service-account: "true"
+    preset-azure-cred: "true"
+    preset-dind-enabled: "true"
+    preset-azure-windows: "true"
+    preset-k8s-ssh: "true"
+    preset-windows-private-registry-cred: "true"
+    preset-windows-repo-list-master: "true"
```

Comment:
I got these preset labels from the windows prow jobs configuration. @chewong Could you confirm if the windows preset labels I've here are required? 

- `preset-azure-windows: "true"`
-  `preset-k8s-ssh: "true"`
-  `preset-windows-private-registry-cred: "true"`
-  `preset-windows-repo-list-master: "true"`

Reply:
LGTM